src/recipe/test/e2e/updateRecipe.controller.e2e-spec.ts

```diff
@@ -66,7 +67,7 @@ describe('Recipe Controller - Update', () => {
   ];
   beforeAll(async () => {
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [RecipeModule, AuthModule],
+      imports: [RecipeModule, AuthModule, AppModule],
```

Comment:
I assume that AppModule imports RecipeModule, AuthModule
Do you need to import Recipe and Auth modules separately?